### ISSUE TITLE
More accessible/contrasting colors dark mode

### ISSUE THREAD

**Issue by Cosmatevs** [aradzie/keybr.com#320]
The colors in the dark mode are too similar to each other to provide comfortable contrast.
It's especially problematic for buttons, lists, and keyboard keys:

![image](https://github.com/user-attachments/assets/bda8665a-f959-4053-9e1a-8a3431327bd7)
![image](https://github.com/user-attachments/assets/8a83d085-db14-4130-bfce-5151c712e81d)
_source: https://webaim.org/resources/contrastchecker/_

**Comment by aradzie** [comment]
I am always receiving conflicting opinions on the topic of themes.
Some people want more contrast, some less.

At least with the custom themes you can have your own colors.

![image](https://github.com/user-attachments/assets/e7a4190d-e52c-470b-917c-63d6379952e1)

![image](https://github.com/user-attachments/assets/9221300d-287b-4efe-806b-2676ee09b8ef)

**Comment by aradzie** [comment]
However, I definitely will take a look at WebAIM. That is an interesting resource, thanks for pointing out.

**Comment by Cosmatevs** [comment]
Yup, I use a custom theme. However, one of the recent updates broke it and I had to recreate it from the dark mode.
It would be much nicer to have a more contrasting and accessible dark theme by default. At least as an additional theme.
